server/src/services/gemini.service.ts

```diff
@@ -108,6 +108,59 @@ Format the output as markdown with the title as an H1 heading.`;
   }
 }
 
+/**
+ * Generate a detailed blog post from a single GitHub commit
+ */
+export async function generateBlogPostFromSingleCommit(
+  commit: GitHubCommit,
+  owner: string,
+  repo: string
+): Promise<string> {
+  if (!config.llm.geminiApiKey) {
+    throw new AppError(500, 'Gemini API key is not configured');
+  }
+
+  // Extract commit details
+  const commitMessage = commit.commit.message;
+  const author = commit.commit.author.name;
```

Comment:
```suggestion
  const { message: commitMessage, author: { name: author } } = commit.commit;
```
이렇게 구조해체할당 문법을 통해서 표현하는 방법도 있답니다!